### PR TITLE
Update amend to set the current date on the commit.

### PR DESCRIPTION
**Pull request by blainesch**
When amending a commit this will update the date of the commit to be the current date.

### CODE REVIEW

**Comment by croaky** [comment]
Cool idea. That seems like it creates it a more accurate commit. I like it.

For `sh` lines, we've been moving them out of aliases into `bin`: https://github.com/thoughtbot/dotfiles/tree/master/bin

**Comment by blainesch** [comment]
@croaky I'll update the commit

**Comment by blainesch** [comment]
Making a pr is difficult when are you using the dotfiles. Heh, if you reset the branch to be on `thoughtbot/master` you lose your local files.

I ended up rebasing my current branch to only have a single commit, then soft resetting it and only pushing the things necessary for this.

Updated! [:

File: bin/git-ca [hunk 1]
```diff
@@ -0,0 +1 @@
+git commit --amend -v --date=$(date +%Y-%m-%dT%H:%M:%S)
```
Comment (gylaz) [inline bin/git-ca]: Thoughts on using `--date="$(date -R)"`? [Source](http://stackoverflow.com/a/9110424/373522)

Comment (blainesch) [inline bin/git-ca]: It doesn't seem to work for me.
~~~
▶ date -R
date: illegal option -- R
~~~

Comment (blainesch) [inline bin/git-ca]: Somebody also commented that `-R` doesn't work in OSX. Saying an alternative is:
~~~
git commit --amend --date="$(date +%s)"
~~~

Comment (gylaz) [inline bin/git-ca]: Apparently, it's a Linux thing. Never mind :smile:

**Comment by gylaz** [comment]
Thanks @blainesch.

**Pull request merged**